src/radical/saga/task.py

```diff
@@ -94,6 +94,11 @@ def __init__ (self, _adaptor, _method_type, _method_context, _ttype) :
         self._attributes_set_getter(c.STATE,     self.get_state)
         self._attributes_set_setter(c.STATE,     self._set_state)
 
+        self._attributes_register  (c.ID,        None,    satt.STRING, satt.SCALAR, satt.READONLY)
```

Comment:
there is no `ID` in constants, that constant was in `radical.saga.job.constants`